opnode/eth/heads.go

```diff
@@ -0,0 +1,48 @@
+package eth
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/event"
+)
+
+type HeadSignal struct {
+	Parent BlockID
+	Self   BlockID
+}
+
+// HeadSignalFn is called to inform a chain with a new head
```

Comment:
Not sure what is meant by "inform a chain with a new head"? Does it just mean a new L1 head was detected?

Reply:
typo, missing word after renaming things, my bad